tests/unit/test_models.py

```diff
@@ -390,6 +407,23 @@ def test_non_converged_solution(self):
             assert np.allclose(output.get(key, [])[0], output.get(key, []))
             assert np.allclose(output_S1.get(key, [])[0], output_S1.get(key, []))
 
+    @pytest.mark.unit
+    def test_set_initial_state(self):
```

Comment:
It would be nice to have assertions for the `model.parameter_set` concentrations as well, or any additional asserts that make sense to verify the built model is correct.

Reply:
I have added an assertion on a third prediction based on the initial OCV value.